tests/ui/type/pattern_types/const_block.stderr

```diff
@@ -0,0 +1,72 @@
+error[E0391]: cycle detected when evaluating type-level constant
```

Comment:
i guess the answer is: we do evaluate them during typeck and just get cycle errors for now :3